Exp-1/subject_results/group.retest/subj.sub_03/output.proc.sub_03

```diff
@@ -0,0 +1,936 @@
+(version 5.15, April 25, 2017)
+echo execution started: `date`
+date
+execution started: Fri Aug  4 12:45:35 EDT 2017
+afni -ver
+Precompiled binary macosx_10.7_local: Jul 27 2017 (Version AFNI_17.2.05)
+afni_history -check_date 23 Sep 2016
+-- is current: afni_history as new as: 23 Sep 2016
+               most recent entry is:   27 Jul 2017
+if ( 0 ) then
+if ( 0 > 0 ) then
+set subj = sub_03
+endif
+set output_dir = sub_03.results
+if ( -d sub_03.results ) then
+set runs = ( `count -digits 2 1 1` )
+count -digits 2 1 1
+mkdir sub_03.results
+mkdir sub_03.results/stimuli
+cp /Users/mli/Desktop/playground/timing_sub-03_ses-retest_Correct_Task.1D /Users/mli/Desktop/playground/timing_sub-03_ses-retest_Incorrect_Task.1D /Users/mli/Desktop/playground/timing_sub-03_ses-retest_No_Response_Control.1D /Users/mli/Desktop/playground/timing_sub-03_ses-retest_No_Response_Task.1D /Users/mli/Desktop/playground/timing_sub-03_ses-retest_Response_Control.1D sub_03.results/stimuli
+3dcopy /Users/mli/Desktop/playground/sub-03/ses-retest/anat/sub-03_ses-retest_T1w.nii.gz sub_03.results/sub-03_ses-retest_T1w
+++ 3dcopy: AFNI version=AFNI_17.2.05 (Jul 27 2017) [64-bit]
+3dTcat -prefix sub_03.results/pb00.sub_03.r01.tcat /Users/mli/Desktop/playground/sub-03/ses-retest/func/sub-03_ses-retest_task-linebisection_bold.nii.gz[0..$]
+++ 3dTcat: AFNI version=AFNI_17.2.05 (Jul 27 2017) [64-bit]
+++ elapsed time = 0.8 s
+set tr_counts = ( 238 )
+cd sub_03.results
+touch out.pre_ss_warn.txt
+foreach run ( 01 )
+3dToutcount -automask -fraction -polort 4 -legendre pb00.sub_03.r01.tcat+orig
+++ 3dToutcount: AFNI version=AFNI_17.2.05 (Jul 27 2017) [64-bit]
+[7m*+ WARNING:[0m   If you are performing spatial transformations on an oblique dset, 
+  such as ./pb00.sub_03.r01.tcat+orig.BRIK,
+  or viewing/combining it with volumes of differing obliquity,
+  you should consider running: 
+     3dWarp -deoblique 
+  on this and  other oblique datasets in the same session.
+ See 3dWarp -help for details.
+++ Oblique dataset:./pb00.sub_03.r01.tcat+orig.BRIK is 4.947676 degrees from plumb.
+++ 18031 voxels passed mask/clip
+if ( `1deval -a outcount.r$run.1D"{0}" -expr "step(a-0.4)"` ) then
+1deval -a outcount.r01.1D{0} -expr step(a-0.4)
+echo ** TR #0 outliers: possible pre-steady state TRs in run 01
+endif
+end
+cat outcount.r01.1D
+foreach run ( 01 )
+3dTshift -tzero 0 -quintic -prefix pb01.sub_03.r01.tshift pb00.sub_03.r01.tcat+orig
+++ 3dTshift: AFNI version=AFNI_17.2.05 (Jul 27 2017) [64-bit]
+[7m*+ WARNING:[0m   If you are performing spatial transformations on an oblique dset, 
+  such as ./pb00.sub_03.r01.tcat+orig.BRIK,
+  or viewing/combining it with volumes of differing obliquity,
+  you should consider running: 
+     3dWarp -deoblique 
+  on this and  other oblique datasets in the same session.
+ See 3dWarp -help for details.
+++ Oblique dataset:./pb00.sub_03.r01.tcat+orig.BRIK is 4.947676 degrees from plumb.
+[7m*+ WARNING:[0m dataset is already aligned in time!
```

Comment:
Note warning that no slice timing was applied. You need to add the correct -tpattern alt+z option in the processing script